CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.16.0-rc.0 / 2020-01-30
+
+* [CHANGE] PromQL: Remove MaxConcurrent from the PromQL engine opts #6712
```

Comment:
It is a bug fix: Queries can time out when queing again

Reply:
And it's an internal change that is not user visible, so shouldn't be listed.at all

Reply:
Before this commit is was not possible to trigger time outs when querying Prometheus even with Max concurrency 1. I think it might surprise some users, which can now get time outs.

Reply:
That's a bug fix, however this particular change as described is pure internal.

Reply:
[BUGFIX] PromQL:Respect query.timeout even when we reach query.max-concurrency #6712

?